tests/installation/partitioning_raid.pm

```diff
@@ -202,12 +202,12 @@ sub modify_uefi_boot_partition {
     # fold the drive tree
     send_key 'left';
     send_key 'left' if is_storage_ng;    # With storage-ng first press folds vdd
-    assert_screen 'partitioning_raid-hard_disks-unfolded';
```

Comment:
Why would you remove this? By that we do not have synchronisation with the above and the below key presses anymore

Reply:
@okurz, we cannot check screen 'unfolded' since it is not available anymore.

Reply:
Well, how do we solve the synchronisation issue then?

@SergioAtSUSE @rwx788 @mloviska @jknphy ?

Reply:
I'm not sure why we have this issue on arm and not on x86_64 factory. So I would say if it's available there, it will be available on arm too. Also not sure what do you mean by it's not available anymore?  partitioning_raid-hard_disks-unfolded-20181219.png is pretty recent and I'm pretty sure one can unfold tree with hard disks.

Reply:
@rwx788 This is inside ```modify_uefi_boot_partition```, only called by aarch64. That is why it was not noticed before.

```partitioning_raid-hard_disks-unfolded-20181219.png``` is used at the very beginning of the test, not at the end where partitions are already done.

Reply:
So, the main problem is that we need to sync key presses. On arm it's even bigger issue than on other architectures. For arm we have a bit weird flow that we create all partitions and then go back to the first one if I recall the code correctly. But I still don't get why are you saying that screen is not available anymore? It asserts that we have disks unfolded and can select first disk from there.

Reply:
Pressing many keys without asserting state has proven to be flaky. The screen you are referring to is exactly what we need. Needle asserts exactly what is on the screen you've pasted. Reason why we don't have similar needle for openSUSE is same as below, test was never executed on ARM.

Reply:
And yes, I know needles are confusing and there is quite some mess with them when we try to reduce the number by reusing some.

Reply:
Let me know if you need some help, I can create new needle for TW on arm. Also in general I guess no changes in the code will be required in the end.

Reply:
@rwx788 ok, so let's re-add the 2 assert_screen. I will make the matching needles in openQA directly.

Reply:
Great, I will take a look later as I really believe that there is no difference to SLE 15 SP1 or aarch64 atm and code should simply work (given correct needles are there).

Reply:
@rwx788 at least the FAT part is needed, as it does not work in Tumbleweed.

Reply:
@rwx788 it looks like SLE select UEFI type, and then type 'fat', whereas here we select directly 'fat' type.